homeworks/vladislav.buzunar_buzyaka/5-arrays/arrays.js

```diff
@@ -0,0 +1,38 @@
+function christmasTree(height) {
+    let branchs;
+    let needlesOnTree;
+    branchs = '';
+    needlesOnTree = 1;
```

Comment:
Extra assignment. Please, move in into `for` as well

Reply:
Here I don't quite understand what needs to be done...